proto/ignition/msgs/parameter_name.proto

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+
+/// \brief Parameter name, used as a request to get a parameter.
+message ParameterName
+{
+  /// \brief Parameter name.
+  // TODO(ivanpauno): Maybe use `stringmsg.proto` instead (?).
```

Comment:
Sorry, what I meant here is not clear.
My idea is that maybe we could just delete this whole file, and instead of having a `ignition::msgs::ParameterName` type use `ignition::msgs::StringMsg`.

Reply:
Fixed